app/api/v1/routes/blockchain.py

```diff
@@ -391,11 +391,70 @@ def getNFTBox(tokenId: str, allowCached=False):
             if len(items) == 1:
                 return items[0]
             else:
-                logging.error(f'ERR:{myself()}: multiple nft box ({e})')
+                logging.error(f'ERR:{myself()}: multiple nft box or tokenId doesn\'t exist')
     except Exception as e:
         logging.error(f'ERR:{myself()}: unable to find nft box ({e})')
         return None
 
+# GET unspent boxes by token id direct from explorer db
+def getUnspentStakeKeyTokenBoxes():
```

Comment:
should be called getUnspentStakeBoxes. Stake keys are unique. What we are looking for here are stake tokens that all have the same token id and identify a legit stake box.